dockers/docker-orchagent/base_image_files/swssloglevel

```diff
@@ -7,4 +7,37 @@ if [ -t 1 ] ; then
     DOCKER_EXEC_FLAGS+="t"
 fi
 
-docker exec -$DOCKER_EXEC_FLAGS swss swssloglevel "$@"
+function help()
+{
+    echo -e "Usage: $0 -n [0 to $(($NUM_ASIC-1))], where n is the namespace instance" 1>&2; exit 1;
```

Comment:
Wondering if it would be good to add something like [OPTIONS] in this help message? I feel it might be confusing that this help suggest using `swssloglevel -n [0 to n]` when inputting `swssloglevel`, and it shows another completely different help message without a `-n` option with `swssloglevel -n 0`

Reply:
Good suggestion, I have updated the help string so user knows there are more options.